ARTICLE-1.markdown

```diff
@@ -2,51 +2,35 @@
+(e) The "Membership Circle" is a committee of CoLab members that is tasked with membership-related policies and procedures and tracking membership statuses.
 
-(f) "Equity Investment" refers to the capital contribution made by the individual to CoLab as part of becoming invested menber as explained in section 1.4 of these Bylaws.
-
-### Section 1.2 Prospective Membership
-
-(a) Benefits
-
-Prospective Members: Prospective Members are persons who may begin the process for becoming a Working Member. They are invited to attend CoLab Summits, can access Board reports and minutes, not including privacy, financial, and legal reports and related confidential issues. They are not yet full Members of the Cooperative.
-
-(b) Qualifications
-
-Persons wishing to become Prospective Members must have worked for CoLab for at least 600 hours in the past year and 300 hours of work in the prior 6 months (an average of 12.5 hours / week), at CoLab Standard Rates.  The Board may waive any of these requirements. Board decisions around qualifications may be appealed directly in writing and approved by full consensus of the Board.
-
-(c) Acceptance
-
-The Team Org Area will track Friends of CoLab and will notify those persons and the Board when they fulfill the qualifications for Prospective Membership. Unless the Board votes to block, the member will be promoted after meeting these qualifications.
-
-### Section 1.3 Working Membership
+### Section 1.2 Future Membership
 
 (a) Benefits
 
-Working Members: Working Members may begin the process for becoming an Invested Member by starting the “Investor Member Candidacy”. Working Members can serve as Org Leads and have all the benefits of Prospective Membership.
+Future Members are invited to attend CoLab Summits and can access Board reports and minutes, not including privacy, financial, and legal reports and related confidential issues. They are not yet Members of the Cooperative.
 
 (b) Qualifications
 
-Persons wishing to become Working Members must have been a Prospective Member for one year, at CoLab Standard Rates. The Board may waive any of these requirements. Board decisions around qualifications may be appealed directly in writing and approved by full consensus.
+Future Members must work for CoLab as employees or contractors for less than one year.
 
 (c) Acceptance
 
-The Team Org Area will track Prospective Members and will notify those persons and the Board when they fulfill the qualifications for Working Membership. Unless the Board votes to block, the Prospective Member will be promoted to Working Member after meeting these qualifications.
+A circle or org area will be tasked with tracking Future Members and will notify those persons and the Board when they fulfill the qualifications for Membership. Unless the Board votes to block, the member will be promoted after meeting these qualifications.
```

Comment:
If we're going to define the "Membership Circle" in section 1.1 above, we might as well refer to this as "Membership Circle" here, I think.

Reply:
I took it out above.